### ISSUE TITLE
Winrm-plugin doesn't work after upgrade rundeck from 4.8.0 to 4.10.0

### ISSUE THREAD

**Issue by krajster** [rundeck/rundeck#8148]
**Describe the bug**
Winrm-plugin (overthere-winrm)(1.3.8) doesn't work after upgrade Rundeck from 4.8.0 to 4.10.0

**My Rundeck detail**

* Rundeck version: [4.10.0]
* install type: [war]
* OS Name/version: [Windows 2019]
* DB Type/version: [mssql ]

**To Reproduce**
1. Go to 'command'
2. Select node with winrm-plugin
3. Run command

**Expected behavior**
 **ERROR services.PluginService - NodeExecutor: configuration was not valid for plugin 'overthere-winrm': hostnameVerify: Invalid value: BROWSER_COMPATIBLE
certTrust: Invalid value: STRICT**

**Screenshots**
![image](https://user-images.githubusercontent.com/8490440/219063599-14e0e2aa-7e16-460e-be10-4693cae8f811.png)

**Desktop (please complete the following information):**
 - OS: [Windows 10]
 - Browser [firefox]
 - Version [109.0.1]

**Additional context**
jvm
implementationVersion : 25.361-b09
name : Java HotSpot(TM) 64-Bit Server VM
vendor : Oracle Corporation
version : 1.8.0_361

change settings:
      "winrm-hostname-trust": "all",
      "winrm-cert-trust": "all"
the problem is not solved.

**Comment by krajster** [comment]
I will add details.
If not set this parameter:
![image](https://user-images.githubusercontent.com/8490440/219127521-1d7a2156-38b3-49db-941d-7c98b1508f10.png)
Seen error:
![image](https://user-images.githubusercontent.com/8490440/219128547-3312fe05-0a07-432e-bb76-e720534176ef.png)

But if set:
![image](https://user-images.githubusercontent.com/8490440/219131001-1263eb04-aea1-4c23-aa77-ebfee45b807f.png)
winrm-plugin works fine.

**Comment by mazzella-c** [comment]
Same issue here and same workaround for the moment.

**Comment by krajster** [comment]
#8105

**Comment by liberodark** [comment]
You have try last 4.10.1 release ?

**Comment by krajster** [comment]
> You have try last 4.10.1 release ?

No, version 4.10.0 is available for me

**Comment by liberodark** [comment]
Yes but is available on Linux for me : `rundeck  noarch  4.10.1.20230221-1`

**Comment by krajster** [comment]
I found the release at https://packagecloud.io/pagerduty/rundeck/.
But I see the same problem:
[2023-02-22T15:49:01,077] ERROR services.PluginService - NodeExecutor: configuration was not valid for plugin '**overthere-winrm': hostnameVerify: Invalid value: BROWSER_COMPATIBLE
certTrust: Invalid value: STRICT**

**Comment by w00dst0ck** [comment]
overthere-winrm is a different plugin

The pywinrm plugin will work

**Comment by MegaDrive68k** [comment]
Hi,

The overthere-winrm is an unmaintained plugin, please use the pywinrm plugin, it is out of the box and is under active development right now.

Greeting!
